frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -107,14 +120,14 @@ function ReaderDictionary:cleanSelection(text)
     return text
 end
 
-function ReaderDictionary:onLookupStarted(word)
+function ReaderDictionary:onLookupInfoStarted(word)
```

Comment:
Since you are changing this file, I would suggest to rename these functions to avoid a conflict with events. Since they are private functions, _showLookupInfo / _hideLookupInfo would be better.